app/containers/DatasetPage.js

```diff
@@ -321,6 +458,15 @@ class DatasetPage extends Component {
                 fullWidth={true} margin="normal" variant="outlined"
               />
           </FlexBox>
+          <FlexBox p={1}>
+            <CpacpySchedulerSelector
+              name='scheduler'
+              buttonProps={{
+                variant: 'contained',
+                className: classes.selectorButton,
+              }}
+            />
```

Comment:
I found that there's a `handleSelect` method in `containers/SchedulerSelector.js`, which will be triggered if one scheduler is selected (in old versions, however, the action that sets the selected scheduler as the current scheduler won't be dispatched by this method). 
Thus, I added the action `CPACPY_SCHEDULER_SCHEDULER` (in `selectCurrentScheduler`) into `handleSelect` in `containers/SchedulerSelector.js`, and didn't enable the `onSelect` attributed in the `CpacpySchedulerSelector` instance. 
Do you think this will work? Will it cause any trouble?

Reply:
It can cause trouble for different scenarios since there is not supposed to have a 'global scheduler' (e.g. a page with two scheduler selectors). Let's keep it this way while we do not have these scenarios.